user/serializers.py

```diff
@@ -2,0 +1,27 @@
+from django.contrib.auth import get_user_model
+from rest_framework import serializers
+
+from user.models import User
+
+
+class UserSerializer(serializers.ModelSerializer):
+    class Meta:
+        model = get_user_model()
+        fields = ("id", "username", "email", "password", "is_staff")
+        read_only_fields = ("id", "is_staff")
+        extra_kwargs = {
+            "password": {"write_only": True, "min_length": 5}
+        }
+
+    def create(self, validated_data):
+        return get_user_model().objects.create_user(**validated_data)
```

Comment:
Ensure that the `create_user` method in your custom user model is correctly implemented to handle password hashing. This is crucial for security.